benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -545,27 +582,6 @@ msgctxt "image alt text"
 msgid "core.icons.idcardcheck"
 msgstr "Icono de tarjeta de identificación con marca de verificación"
 
-msgid "eligibility.pages.start.login_gov.required_items[0]"
-msgstr "Su tarjeta de identificación emitida por el estado"
-
-msgid "eligibility.pages.start.login_gov.required_items[1]"
-msgstr "Su número de Seguro Social"
-
-msgid "eligibility.pages.start.login_gov.required_items[2]"
-msgstr "Un número de teléfono con un plan de teléfono asociado con su nombre"
-
-msgid "eligibility.pages.start.veteran.required_items[0]"
-msgstr "Login.gov"
-
-msgid "eligibility.pages.start.veteran.required_items[1]"
-msgstr "ID.me"
-
-msgid "eligibility.pages.start.veteran.required_items[2]"
-msgstr "DS Logon"
-
-msgid "eligibility.pages.start.veteran.required_items[3]"
-msgstr "My HealtheVet"
-
```

Comment:
???? Have no idea why Django did this !!!!!

Reply:
I know why now 😓 thankfully the changes in #1498 will prevent this clobbering from happening in the future by `bin/makemessages.sh` b/c the `msgid`s will be in the app.